src/firewall/core/io/zone.py

```diff
@@ -654,6 +655,18 @@ def startElement(self, name, attrs):
             if "priority" in attrs:
                 priority = int(attrs["priority"])
             self._rule = rich.Rich_Rule(family=family, priority=priority)
+        
+        elif name == "tcp-mss-clamp":
+            value = None
+            if "value" in attrs:
+                value = attrs["value"]
+                if value not in ["pmtu"] or not value.isdigit()
+                    log.warning('Invalid tcp-mss-clamp: Value "%s" invalid',
+                                attrs["value"])
+                    self._rule_error = True
+                    return
+             self._rule = rich.Rich_Tcp_Mss_clamp(value=value)        
```

Comment:
This whole bit needs to be guarded by `if self._rule` as tcp-mss-clamp is only valid in rich rules. It's not a zone level feature.